server/cmd/genkey/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/decred/dcrd/dcrec/secp256k1/v2"
+)
+
+func actualMain() error {
+	secPrivKey, err := secp256k1.GeneratePrivateKey()
+	if err != nil {
+		return err
+	}
+	// privkeyFile, err := os.OpenFile("dexprivkey.pem", os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0600)
+	// if err != nil {
+	// 	return err
+	// }
+	// defer privkeyFile.Close()
+
+	// err = pem.Encode(privkeyFile, &pem.Block{
+	// 	Type:  "EC PRIVATE KEY",
+	// 	Bytes: secPrivKey.Serialize(),
+	// })
+	// if err != nil {
+	// 	return err
+	// }
+
+	err = ioutil.WriteFile("dexprivkey", secPrivKey.Serialize(), 0600)
```

Comment:
I think you mean "this key encoding scheme is garbage" because it is, and just enough to move forward on the server dev.  We will have to settle on a key encoding scheme, and that's a helpful link, although I'm not entirely sure if we want to require a password on startup.